plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -23,3 +23,13 @@ if [ -f /usr/libexec/pk-command-not-found ]; then
         return $retval
     }
 fi
+
+# OSX command-not-found support
+# https://github.com/Homebrew/homebrew-command-not-found
+if [ -e /usr/local/bin/brew ]; then
+  if brew tap | command grep -q homebrew/command-not-found; then
```

Comment:
This check is to ensure that the homebrew/command-not-found tap has been installed, not to check the actual command being run which is done in the next line.